nwbwidgets/analysis/placefields.py

```diff
@@ -129,7 +136,8 @@ def compute_2d_firing_rate(pos, pos_tt, spikes,
                            speed_thresh=0.03,
                            gaussian_sd=0.0184,
                            x_start=None, x_stop=None,
-                           y_start=None, y_stop=None):
+                           y_start=None, y_stop=None,
+                           velocity=[]):
```

Comment:
An optional variable usually takes default value `None` and that's what should be used here.